env.go

```diff
@@ -0,0 +1,52 @@
+package odbc
+
+import (
+	"github.com/mpcjanssen/odbc/api"
+	"unsafe"
+)
+
+type DSN struct {
+	Name string
+	Description string
+}
+
+func DataSources() ([]DSN, error) {
```

Comment:
This needs documentation explaining what it does. Same for DSN.

And commit description needs to explain what is added and why it is useful.

And I am not sure that it is useful. So, please, discuss what you are trying to achieved before investing more time into this.

Reply:
I am building a REST-ish proxy to an ODBC database. In order to test the proxy I made a small web form frontend to execute queries. 
The point of this addition is to make it possible to select a DSN from a dropdown box.
I currently don't have the time to go through the pkgname rewriting link you posted (thanks for that though) so for now I will keep this functionality in my own fork.

Reply:
> I currently don't have the time to go through the pkgname rewriting link you posted (thanks for that though) so for now I will keep this functionality in my own fork.

Fair enough. SGTM.

Alex